### ISSUE TITLE
Kaggle Kernel  could not find function "keywords_rake"

### ISSUE THREAD

**Issue by amrrs** [bnosac/udpipe#18]
The function to use `RAKE` works very well on my local machine but when I try to run it on a Kaggle Kernel. It just doesn't work

```term = "lemma", group = "doc_id", relevant = x$upos %in%  : 
  could not find function "keywords_rake"
Calls: render ... handle -> withCallingHandlers -> withVisible -> eval -> eval```

Any possible suggestion?

**Comment by amrrs** [comment]
Actually even the function isn't displayed there:
```
> ls('package:udpipe')
[1] "as_phrasemachine"          "collocation"              
 [3] "cooccurrence"              "document_term_frequencies"
 [5] "document_term_matrix"      "dtm_cbind"                
 [7] "dtm_cor"                   "dtm_rbind"                
 [9] "dtm_remove_lowfreq"        "dtm_remove_terms"         
[11] "dtm_remove_tfidf"          "dtm_reverse"              
[13] "dtm_tfidf"                 "phrases"                  
[15] "txt_collapse"              "txt_freq"                 
[17] "txt_highlight"             "txt_next"                 
[19] "txt_nextgram"              "txt_previous"             
[21] "txt_recode"                "txt_sample"               
[23] "txt_show"                  "udpipe_accuracy"          
[25] "udpipe_annotate"           "udpipe_download_model"    
[27] "udpipe_load_model"         "udpipe_read_conllu"       
[29] "udpipe_train"              "unique_identifier"

```

**Comment by jwijffels** [comment]
the function keywords_rake was released as part of version 0.4 of the udpipe R package. It looks like you have an older version of the udpipe R package on that Kaggle kernel.
To check the version of the package on your 'Kaggle Kernel' use installed.packages()["udpipe", ]

Update the package with install.packages('udpipe') so that you have version 0.4 of the udpipe R package which is on CRAN since February 07 2018

**Comment by amrrs** [comment]
Thanks for the details. I've sent a PR to install this package again. I can't do that in my Kernel code ( tried and it threw an error).
